### ISSUE TITLE
Stream Error 4003

### ISSUE THREAD

**Issue by Thunderbolt8** [fgl27/SmartTwitchTV#204]
Getting a "Stream Error 4003" most of the times I try to access streams "betboom_eng", "betboom_eng2", "betboom_eng3". I say most of the times because I suspect its related to their bitrate settings. it seems to be problematic if their source 1080p60 bitrate is higher than 8.60ish or so. then its either this error message with a black screen or sometimes (tough rarely) a standstill screen while there is sound running. accessing their stream at lower resolutions works normally for me (unless I deactivate the auto reduce quality setting of course).

any idea why this is?

**Comment by fgl27** [comment]
Are you are this happens in the Android version of the app?

This is the Android only repo

The Samsung https://github.com/fgl27/smarttv-twitch

I have see reports like this but not for Android.

I can't replicate also have no workaround for now

**Comment by fgl27** [comment]
Can you replicate the issue in vods? Share a vod that you can?

**Comment by Thunderbolt8** [comment]
yes, it is an AndroidTV (Phillips 4k TV, albeit a really old one >7-8 years old)

Basically the streams from the last 3 days (probably more), e.g.:

https://www.twitch.tv/videos/2032257178
https://www.twitch.tv/videos/2031172112
https://www.twitch.tv/videos/2030099707

**Comment by Thunderbolt8** [comment]
btw I noticed this actually works when I enable the google codec and disable the other one. though "works" is a bit much, the livestream is so slow its basically unwatchable and vods do start, but exit as soon as I try to skip forward.

**Comment by fgl27** [comment]
4003 is codec unsupported

https://github.com/google/ExoPlayer/blob/9c9f5a0599ec012d5cc46e3bd2e732a589adf61d/library/transformer/src/main/java/com/google/android/exoplayer2/transformer/ExportException.java#L157

The device is just out of date.

Google codec is SW codec.

The SW codec isn't affected but because there device is old doesn't have the performance to be used

**Comment by fgl27** [comment]
The only workaround is to disable source for those.

**Comment by fgl27** [comment]
The app doesn't have that level of configuration.

You can disable globally only.

**Comment by Thunderbolt8** [comment]
alright, thank your for your help.
